### ISSUE TITLE
Computer stuck at GRloading...

### ISSUE THREAD

**Issue by dudefoxlive** [ventoy/Ventoy#27]
I have a machine that when attempting to boot gets stuck at a screen that says "GRloading...". What can I do to try and fix this? So far it's only 1 machine. What logs would help?

**Comment by Huckleberry-MX** [comment]
Same here. Tried both Windows and Linux installations on a 2GB usb stick (that always works). Classic Bios, 32bit. Though the installations were successful on both systems, it stays at:

GRloading.. Read Error
____________________________________
I did:

sudo sh Ventoy2Disk.sh -i /dev/sdb

on  Linux and the installation was ok.

**Comment by fchamieh** [comment]
same here, ventoy 1.0.27 on a sandisk cruiser (8GB) usb 2.0 stick, installled from linux. Using it on an old HP Pavillion Dv6 (i7, AMD Radeon 6770, 8GB RAM, 256 Liteon SSD).

**Comment by dudefoxlive** [comment]
Interesting. Mine is also an hp pavilion dv6. Though with an amd cpu.

**Comment by fchamieh** [comment]
> Interesting. Mine is also an hp pavilion dv6. Though with an amd cpu.

Did you manage to resolve this issue?

**Comment by dudefoxlive** [comment]
Sadly no. Its rare that I would need to use it on that machine anyway. If I have to reinstall Windows I use my SCCM server. Still would like to see this get fixed. It could be something with the machine though.

**Comment by Pascal666** [comment]
I too was stuck at the "GRloading..." screen on an HP Pavilion dv7-6135dx.  Solution was to downgrade to v1.0.26.  Both v1.0.27 and v1.0.29 have this bug, v1.0.26, v1.0.22, and v1.0.15 do not.

**Comment by Bjoern75900** [comment]
> I too was stuck at the "GRloading..." screen on an HP Pavilion dv7-6135dx. Solution was to downgrade to v1.0.26. Both v1.0.27 and v1.0.29 have this bug, v1.0.26, v1.0.22, and v1.0.15 do not.

so what....not booting from iso on hp pavilion dv6 anyway....:(

**Comment by ventoy** [comment]
Please use the latest 1.0.38 with a **fresh install** and test again.
Some one told me that Ventoy will stuck at VTLoading if the partition is not aligned with 4KB.
So Ventoy will align the partition by 4KB since 1.0.36

**Comment by Bjoern75900** [comment]
tested alreadyon 3 hp pavilions....1.0.38 too :(
v1.0.26, v1.0.22, and v1.0.15 do not display stuck at the GRloading... screen, but that's all.
none of the isos booots

**Comment by ventoy** [comment]
You mean that with 1.0.38 it will stuck at "VTloading ..." and with 1.0.26 no stuck at GRloading?
Can you boot into Ventoy's menu with 1.0.26?

**Comment by Bjoern75900** [comment]
yes. but only into menu. after i choose any iso, computer hangs on.
on dell laptop it works, i mean ver. 1.0.38

**Comment by Bjoern75900** [comment]
1.0.39 the same :( stuck at 'VTloading ....' 
SHIT

**Comment by topblast** [comment]
I have an HP Pavilion dv7-6c80us and I am getting this error with version 1.0.43.

**Comment by charlesbrumbaugh** [comment]
Same issue here v1.0.44 on a dv7

**Comment by Bjoern75900** [comment]
I'm curious if the author reads it at all.

**Comment by gmcamposano** [comment]
Same here. Using a DV6 Pavilion.
I have 3 computers. Same USB boots on 2 computers, and on HP it gets stuck with VTLoading....

**Comment by TheMuntan** [comment]
Exact same issue here on an hp pavilion dv6 even on 1.0.46 stuck on VTloading

**Comment by Bjoern75900** [comment]
Simply shitty coded app.....

**Comment by Pascal666** [comment]
> Simply shitty coded app.....

This error has only been reported on HP Pavilion dv6 and dv7 laptops.  Both were released over a decade ago and are no longer supported by HP.  Of all the desktops and laptops out there, two related laptops from a single manufacturer give this error.  That is not a "shitty coded app", that is a BIOS bug in those two laptops.

The author may be able to work around that bug at some point.  That may happen faster if someone donates one of those laptops so the author can actually see the problem.

**Comment by Bjoern75900** [comment]
Ok, You are right.

**Comment by rathel** [comment]
This happens on my old Dell desktop if I plug the stick directly, but if I plug it into my USB enabled keyboard it seems to work fine. Just a thought.

**Comment by kulkom** [comment]
Same here on a Pavilion DV7, stuck at "VTloading......" indefinitely. Workaround, anyone?

**Comment by TeoColuccio** [comment]
Same problem of "VTloading......" stuck on HP pavilion dv6

**Comment by Gabgobie** [comment]
Got an old lenovo idaecentre (i7 2600) and ventoy is getting stuck at "VTloading . . . . . . ."

In BIOS the USBs name will also not be displayed but it is recognized as a boot option.
It was on a 256G usb. I tried again with a smaller one (16G) and it worked. I'll reinstall Ventoy on the big USB but size might be the problem here. Also the small USB is still on Ventoy 1.0.95. The 256G USB is on 1.0.97. I did try to downgrade to 1.0.95 but the big one is still stuck on "VTloading . . . . . . .".

I'll assume it's an issue with size for all those who come across this issue.
